src/disclosures/js/views/expenses-view.js

```diff
@@ -73,6 +76,27 @@ var expensesView = {
         value = stringToNum( $ele.val() ),
         key = $ele.attr( 'data-expenses' );
     publish.expensesData( key, value );
+    expensesView.updateView( getExpenses.values() );
+  },
+
+  /**
+   * Helper function to indicate that a offer summary line item has
+   * successfully recalculated
+   * @param {object} element - jQuery object of the recalculated summary element
+   */
+  addSummaryRecalculationMessage: function( element ) {
+    element.siblings().hide();
+    element.text( 'Updating...' );
+  },
+
+  /**
+   * Helper function to remove all indicators that data has recalculated
+   * @param {object} element - jQuery object of the recalculated summary element
+   * @param {string} value - the recalculated value of the element
+   */
+  removeRecalculationMessage: function( element, value ) {
+    element.text( value );
+    element.siblings().show();
```

Comment:
They actually didn't start out as a copy/paste, but it looks like they evolved to be identical. :)

I can fix this, but I don't want it to hold up this PR. Can it be resolved later?